src/Cookie/CookieJar.php

```diff
@@ -98,6 +98,25 @@ public static function shouldPersist(
         return false;
     }
 
+    /**
+     * Finds and returns the cookie based on the name
+     *
+     * @param string $name cookie name to search for
+     * @return SetCookie|null cookie that was found or null if not found
+     */
+    public function getCookieByName($name)
+    {
+        $foundCookie = null;
+        foreach($this->cookies as $cookie) {
+            if(strtolower($cookie->getName()) === strtolower($name)) {
+                $foundCookie = $cookie;
+                break;
+            }
+        }
+
+        return($foundCookie);
```

Comment:
just delete it then, since the method will always return null if nothing else is returned